src/views/minor/Footer.vue

```diff
@@ -1,20 +1,33 @@
 <template>
     <footer class="page-footer font-small">
-        <div class="footer-copyright text-center py-3" style="color: white">
+        <div class="text-center py-3" style="color: white">
             {{ copyright }}
+            <p>
+                Licensed under the
+                <a
+                    class="footer-link"
+                    href="https://opensource.org/licenses/MIT">
+                    MIT License</a
+                >.
+            </p>
+            <p>
+                By using this website, you agree any images you create are
+                released into the public domain.
+            </p>
         </div>
     </footer>
 </template>
 
 <script>
     const currentYear = new Date().getFullYear()
+    // prettier-ignore
     const copyrightText
-        = `Copyright © 2020-${currentYear} University of Colorado Boulder`
+    = `Copyright © 2020-${currentYear} Regents of University of Colorado`
```

Comment:
"Regents of the University of Colorado" (I checked the university's own webpages for exact wording)

Reply:
Should be fixed in https://github.com/numberscope/frontscope/commit/53aa6b1ae8f0d7baa5bddcdba7c69a9fb09fd8a9.